tutorials/notebooks/IMX500_notebooks.md

```diff
@@ -0,0 +1,12 @@
+# Sony-IMX500 Notebooks
+
+Here we provide examples on quantizing pre-trained models for deployment on Sony-IMX500 processing platform.
+We will cover various tasks and demonstrate the necessary steps to achieve efficient quantization for optimal
+deployment performance.
```

Comment:
highlight that the exported models from these tutorials are ready to be deployed on IMX500! (plug-and-play)